components/api/src/main/java/com/hotels/styx/api/HttpResponse.java

```diff
@@ -516,7 +516,7 @@ public Builder addCookie(String name, String value) {
          */
         public Builder removeCookie(String name) {
             cookies.stream()
-                    .filter(cookie -> cookie.name().equalsIgnoreCase(name))
+                    .filter(cookie -> cookie.name().equals(name))
```

Comment:
Now when you are at it, would you mind also fixing any other instances where the `equalsIgnoreCase` is used for cookie names?

I think there would be in the `HttpResponse`, `HttpRequest`, `FullHttpRequest`, `FullHttpResponse`, `StreamingHttpRequest`, and `StreamingHttpResponse` classes.